pipeline.py

```diff
@@ -26,7 +26,7 @@ def to_text(self):
         """
         Converts instance to str format
         """
-        pass
```

Comment:
discard changes in this file

Reply:
i made it due to lint error W0107 (unnecessary-pass), and in line 93 due to W0612 (unused-variable) also

Reply:
discarded changes